### PR TITLE
[docs] Update debezium-server docs

### PR DESCRIPTION
**Pull request by nguymin4**

### CODE REVIEW

File: documentation/modules/ROOT/pages/operations/debezium-server.adoc [hunk 1]
```diff
@@ -207,7 +207,10 @@ The source configuration uses the same configuration properties that are describ
 |[[debezium-source-offset-storage]]<<debezium-source-offset-storage, `debezium.source.offset.storage`>>
 |`org.apache.kafka.connect.storage.FileOffsetBackingStore`
 |Class to use for storing and retrieving offsets for non-Kafka deployments.
-To use Redis to store offsets, use `io.debezium.storage.redis.offset.RedisOffsetBackingStore`
+Available options
+
+* `org.apache.kafka.connect.storage.FileOffsetBackingStore` for non-Kafka deployments  
+* `io.debezium.storage.redis.offset.RedisOffsetBackingStore` for Redis deployments
```
Comment (nguymin4) [inline documentation/modules/ROOT/pages/operations/debezium-server.adoc:213]: I think we can also add `MemoryOffsetBackingStore` and `JdbcOffsetBackingStore` here as well but this is up for discussion.

Comment (Naros) [reply documentation/modules/ROOT/pages/operations/debezium-server.adoc:213]: I have no objection, if you want to add that to the docs as well, :+1:

Comment (nguymin4) [reply documentation/modules/ROOT/pages/operations/debezium-server.adoc]: @Naros I added those two and amended the commit. Thank you.

**Comment by jpechane** [comment]
@nguymin4 Applied, thanks

**Pull request merged**